synthetics/data_source_tests.go

```diff
@@ -38,9 +37,9 @@ func dataSourceTests() *schema.Resource {
 }
 
 func dataSourceTestsRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	tflog.Debug(ctx, "Kentik API request - read tests")
+	tflog.Debug(ctx, "List synthetic Kentik API request")
```

Comment:
```suggestion
	tflog.Debug(ctx, "List Tests Kentik API request")
```
The name of the resource is Test...
In other places too.

Reply:
Done